<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_hum_alarm">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>200</y>
      <width>76</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Hum. Alarm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t_alarm">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>200</y>
      <width>76</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp Alarm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hum_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>200</y>
      <width>55</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>76</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_select">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_up">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_down">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sensor">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>51</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensor</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="screen_output">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>1081</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_convertTemp">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert Temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>379</y>
      <width>551</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Super Dooper Sensor Co.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led_sensor">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="led_temp">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="led_hum">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="led_t_alarm">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="led_h_alarm">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
